draft-age-masque-connect-ip.md

```diff
@@ -350,10 +350,35 @@ forward.
 
 # Examples
 
+CONNECT-IP enables many different use cases that can benefit from IP
+packet proxying and tunnelling. These examples are provided to help
+illustrate some of the ways in which CONNECT-IP can be used.
+
+## Remote Access VPN
+
```

Comment:
Can we also add the term "full-tunnel" somewhere in the preexisting paragraph below? I know it's mentioned under the diagram but I think seeing it in the TLDR for the example will help when scrolling through the examples.

Reply:
I think there's value in duplicating the section, because this is an example. Those two use-cases are distinct in the IP proxying requirements draft, so explicitly showing how two distinct use-cases are handled in stand-alone examples is useful. The fact that the two sections become very close to one another is even a plus: it shows that the protocol design allows component reuse. In other words, I think duplication in the normative text is bad, but not in examples.